packages/perseus/src/widgets/interactive-graphs/mafs-styles.css

```diff
@@ -124,3 +124,8 @@
 .MafsView .movable-point:focus-visible .movable-point-focus-outline {
     visibility: visible;
 }
+
+.MafsView .axis-ticks text {
+    font-size: 14px;
+    font-family: "MJXTEX";
```

Comment:
Where is this font family defined? Will it always be available?

I kind of thought we'd have to add a font-face definition like:

```css
@font-face {
  font-family: MJXTEX;
  src: url("https://cdn.kastatic.org/fonts/mathjax/MathJax_Main-Regular.woff") format("woff");
}
```

But maybe that definition already exists elsewhere in Perseus?

If we don't have an explicit font-face definition, then I think the correct font will only be available if MathJax has rendered an expression since the last full-page refresh.

Reply:
If we end up defining the font-face in Perseus CSS, we should probably call it something other than `MJXTEX` to avoid any possible conflict with the font-face that MathJax defines. They _should_ always be the same font, but you never know...

Reply:
We found that it's actually already well defined in MathJax itself, which I believe gets pulled into index.css when we compile the dist. 

![Screenshot 2024-04-11 at 2 51 24 PM](https://github.com/Khan/perseus/assets/12463099/505ec76e-0436-43bc-8d50-376926081dae)

I made sure to test in webapp, and it loads in properly even if I go directly to the segment graph test page without loading any ancillary MathJax. :)

Reply:
I believe @jeremywiebe thought this would be bundled properly after we did some digging! 

It also seems to load properly in flipbook, even when I make sure that there's no other Mathjax on the page at all.
 
![Screenshot 2024-04-11 at 3 04 31 PM](https://github.com/Khan/perseus/assets/12463099/3f0c7ce9-51e0-46e2-b7e7-d8638d970d02)

I was unable to find any examples of content in webapp that don't _also_ end up using ancillary MathJax as they usually have to indicate points or some kind of value to set up the question. Even the segment testing page has some minor bits of MathJax floating around.🤔

Reply:
It does indeed seem like MathJax sneaks in in a lot of places!

However, I'm still wary of relying on MathJax being the math renderer that is used in production. We allow clients of Perseus to inject any TeX component, and I suspect that third-party users of Perseus (e.g. [Learning Equity](https://khanacademy.slack.com/archives/CJDRXTGQ7/p1712852640716959)) are still using KaTeX.

I tried out switching to a custom `@font-face` and it works. The fonts are only loaded once if we use the same CDN URL that the MathJax renderer is configured to use:

```css
@font-face {
    font-family: Mafs-MJXTEX;
    src: url("https://cdn.kastatic.org/fonts/mathjax/MathJax_Main-Regular.woff") format("woff");
}

.MafsView .axis-ticks text {
    font-size: 14px;
    font-family: "Mafs-MJXTEX";
}
```

Reply:
Lovely, that seems safer! I've implemented your suggestion, thank you greatly. :)